samples/cpp/benchmark_app/README.md

```diff
@@ -79,11 +79,14 @@ Options:
     -h, --help                  Print a usage message
     -m "<path>"                 Required. Path to an .xml/.onnx/.prototxt file with a trained model or to a .blob files with a trained compiled model.
     -i "<path>"                 Optional. Path to a folder with images and/or binaries or to specific image or binary file.
+                                In case of dynamic shapes networks with several inputs provide the same number of files for each input (except cases with single file for any input):
+                                "input1:1.jpg input2:1.bin", "input1:1.bin,2.bin input2:3.bin input3:4.bin,5.bin ".
+                                Also you can pass specific keys for inputs: "random" - for fillling input with random data, "image_info" - for filling input with image size.
```

Comment:
1. BTW how it works for cases when input is not specified directly? Do we have backward compatible behavior for models with image_info? 
2. What about supporting these keywords in python? @AlexeyLebedev1

Reply:
> 1. BTW how it works for cases when input is not specified directly? Do we have backward compatible behavior for models with image_info?

There are two functions now, first one 
https://github.com/openvinotoolkit/openvino/blob/5153af3fd1fee22b8d316ed6b1617435637f983e/samples/cpp/benchmark_app/inputs_filling.cpp#L480
Fillling inputs in old backward-compatible way for static models when there are no names specified.
And https://github.com/openvinotoolkit/openvino/blob/5153af3fd1fee22b8d316ed6b1617435637f983e/samples/cpp/benchmark_app/inputs_filling.cpp#L323
Which fills inputs for dynamic models (in both cases, with and without names) and for static when names are specified.

Reply:
> 2. What about supporting these keywords in python? 

These keywords are quiet optional, I added them just for convenience.